terraform/modules/ecs/_variable.tf

```diff
@@ -0,0 +1,106 @@
+variable "appmesh_name" {
+  type        = string
+  description = ""
+}
+
+variable "availability_zones" {
+  description = "AZs"
+  type        = list(string)
+}
+
+variable "az_count" {
+  description = "Number of AZs to cover in a given region"
+  default     = "2"
+}
+
+variable "cloudmap_namespace_name" {
+  type        = string
+  description = ""
+}
+
+variable "cloudmap_service_name" {
+  type        = string
+  description = ""
+}
+
+variable "cw_retention_in_days" {
+  type = number
+}
+
+variable "ecs_alb_name" {
+  description = "ALB Name"
```

Comment:
What is an ALB? Can we expand this?

Reply:
Should be better now! https://github.com/CDCgov/dibbs-aws/pull/1#discussion_r1669434026